src/main/java/net/glowstone/generator/TheEndGenerator.java

```diff
@@ -135,16 +135,16 @@ private void generateTerrainDensity(World world, int x, int z) {
 
         for (int i = 0; i < 3; i++) {
             for (int j = 0; j < 3; j++) {
-                double noiseHeight = 100.0D - Math.sqrt((x + i) * (x + i) + (z + j) * (z + j)) * 8.0D;
-                noiseHeight = Math.max(-100.0D, Math.min(80.0D, noiseHeight));
+                double nh = 100.0D - Math.sqrt((x + i) * (x + i) + (z + j) * (z + j)) * 8.0D;
```

Comment:
The changes in this file are not correct.

Reply:
Reverted the file.